unit_tests/mocks/services/event_service.py

```diff
@@ -14,46 +14,58 @@ def maybe_throw():
         if event_service.throw_repository_error:
             raise RepositoryError("bad repo")
 
-    def mock_get_all_documents() -> list[EventReadDTO]:
+    def mock_get_all_events() -> list[EventReadDTO]:
```

Comment:
Grief did I really miss that on the last review :laughing:

Reply:
So did I...😅